android/app/build.gradle

```diff
@@ -148,6 +148,7 @@ android {
 dependencies {
     compile project(':react-native-linear-gradient')
     compile project(':magnet-scanner-android')
+    compile project(':reactnativemapboxgl')
```

Comment:
Also I tried the react-native-maps by Airbnb that the React Native team recommends, but it ended up being a massive mess to set up and configure and it isn't even compatible with our Google Play Service version. So falling back to mapbox was the best solution.